mypy/semanal.py

```diff
@@ -1801,6 +1801,9 @@ def process_newtype_declaration(self, s: AssignmentStmt) -> None:
             self.fail(message.format(old_type), s)
             return
 
+        if 'unimported' in self.options.disallow_any and has_any_from_unimported_type(old_type):
```

Comment:
Okay, I think it's a very good point. 
I will implement this in a different PR.